shared/src/main/java/bmw/ximalaya/test/media/BrowseTree.kt

```diff
@@ -1,76 +1,109 @@
                     flag = MediaBrowserCompat.MediaItem.FLAG_BROWSABLE
                 }.build()
             )
-        ), Pair(NEU_HOME_ROOT, mutableListOf())
-        , Pair(NEU_BROWSER_ROOT, mutableListOf())
-        , Pair(NEU_RECENT_ROOT, mutableListOf())
-        , Pair(NEU_LIBRARY_ROOT, mutableListOf())
+        ), Pair(TINGYU_HOME_ROOT, mutableListOf())
+        , Pair(TINGYU_BROWSER_ROOT, mutableListOf())
+        , Pair(TINGYU_RECENT_ROOT, mutableListOf())
+        , Pair(TINGYU_LIBRARY_ROOT, mutableListOf())
     )
 
     operator fun get(key: String) = mediaIdToChildren[key]
     fun init() {
-        musicSource.load()
+        musicSource.load(xmlyPlayer)
         musicSource.whenReady {
             if(it){
                 val sourceList = musicSource.map { it }
-                this[NEU_BROWSER_ROOT]?.add(
+                this[TINGYU_BROWSER_ROOT]?.add(
                     MediaMetadataCompat.Builder().apply {
-                        id = NEU_BROWSERLEVEL1_ROOT
-                        title = "BROWSERLEVEL1"
+                        id = "id_jdlg"
+                        title = "经典老歌"
+                      //  artist = "周华健"
+                        albumArtUri = sourceList[0].albumArtUri.toString()
                         flag = MediaBrowserCompat.MediaItem.FLAG_BROWSABLE
                     }.build()
                 )
-               // mediaIdToChildren["id_jdlg"] = mutableListOf<MediaMetadataCompat>().apply{addAll(sourceList)}
-                mediaIdToChildren[NEU_BROWSERLEVEL1_ROOT] = mutableListOf<MediaMetadataCompat>().apply{
+                mediaIdToChildren["id_jdlg"] = mutableListOf<MediaMetadataCompat>().apply{
                     this?.add(
                         MediaMetadataCompat.Builder().apply {
-                            id = "id_jdlg"
-                            title = "经典老歌"
+                            id = "id_jdlgex"
+                            title = "热门歌曲"
                             albumArtUri = sourceList[0].albumArtUri.toString()
                             flag = MediaBrowserCompat.MediaItem.FLAG_BROWSABLE
-                        }.build())
+                        }.build()
+                    )
+                }
+//                mediaIdToChildren["id_jdlg"] = mutableListOf<MediaMetadataCompat>().apply{addAll(sourceList)}
+                mediaIdToChildren["id_jdlgex"] = mutableListOf<MediaMetadataCompat>().apply{
+                    addAll(sourceList)
                 }
-                mediaIdToChildren["id_jdlg"] = mutableListOf<MediaMetadataCompat>().apply{addAll(sourceList)}
-                this[NEU_HOME_ROOT]?.addAll(sourceList)
-                this[NEU_RECENT_ROOT]?.addAll(sourceList)
-                this[NEU_LIBRARY_ROOT]?.addAll(sourceList)
+
+                NeuLog.e("mediaItems(${musicSource.albumList})")
+                this[TINGYU_HOME_ROOT]?.addAll(
+                    musicSource.albumList
+//                    getMediaMetadataCompats()
```

Comment:
Remove the useless code

Reply:
Need to keep during debugging, will be removed before official released.